src/app/requests/CreateRequest.ts

```diff
@@ -24,18 +26,21 @@ export class CreateRequest extends Request {
     constructor(args: {
         commandContext: CommandContext, 
     }) {
+        Log.debug(`In CreateRequest`);
         if (args.commandContext) {
             const commandContext: CommandContext = args.commandContext;
-            if (commandContext.subcommands[0] !== Activities.create) {
+            const isIOU = commandContext.commandName == 'iou' ? true : false;
+            if (!isIOU && (commandContext.subcommands[0] !== Activities.create)) {
                 throw new Error('CreateRequest attempted for non Create activity.');
             }
-            const isIOU = commandContext.commandName == 'iou' ? true : false;
-            super(commandContext.subcommands[0], commandContext.guildID, args.commandContext.user.id, args.commandContext.user.bot);
+            super(Activities.create, commandContext.guildID, args.commandContext.user.id, args.commandContext.user.bot);
+            this.createdInChannel = commandContext.channelID;
             this.userId = commandContext.user.id;
             if (isIOU) {
-                this.title = commandContext.options.create['why'];
-                this.owedTo = commandContext.options.create['owed-to'];
+                this.title = commandContext.options['why'];
+                this.owedTo = commandContext.options['owed-to'];
```

Comment:
Perhaps another good 'first issue' is to refactor all the hard coded literals in `commandContext.options[]` throughout our `*Request` classes to constants

Reply:
Yes